app/validators/profile_validator.rb

```diff
@@ -40,7 +40,7 @@ def summary_too_long?(record)
     return if record.summary.blank?
 
     # Grandfather in people who had a too long summary before
-    previous_summary = record.data_was[SUMMARY_ATTRIBUTE]
+    previous_summary = record.data_was[SUMMARY_ATTRIBUTE] || record.summary_was
```

Comment:
Will we eventually remove the `record.data_was[SUMMARY_ATTRIBUTE` bit?

Reply:
Yes and I actually realized this should be swapped because we want to prioritize the `summary` column over the `data` column in cases where both exist. Thanks for pointing this out.